synapse/rest/client/v2_alpha/account.py

```diff
@@ -630,34 +634,43 @@ def on_POST(self, request):
         # Try to validate as email
         if self.hs.config.threepid_behaviour_email == ThreepidBehaviour.REMOTE:
             # Ask our delegated email identity server
-            validation_session = yield self.identity_handler.threepid_from_creds(
-                self.hs.config.account_threepid_delegate_email, threepid_creds
-            )
+            try:
+                validation_session = yield self.identity_handler.threepid_from_creds(
+                    self.hs.config.account_threepid_delegate_email, threepid_creds
+                )
+            except HttpResponseException:
+                pass
```

Comment:
we should probably at least log something here?

Reply:
Yeah, we can do a debug log?

Reply:
i'd suggest info

Reply:
Have done a debug log.